packages/core-bridge/src/lib.rs

```diff
@@ -172,13 +173,13 @@ fn tokio_runtime() -> &'static Runtime {
     })
 }
 
-/// Send a result to JS via callback using an [EventQueue]
-fn send_result<F, T>(queue: Arc<EventQueue>, callback: Root<JsFunction>, res_fn: F)
+/// Send a result to JS via callback using an [Channel]
```

Comment:
```suggestion
/// Send a result to JS via callback using a [Channel]
```

Reply:
My search and replace didn't catch this :)